<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icare</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 232, 250);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_serial">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开启串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_touch">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>无触摸</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>触摸传感器</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_fire">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>无烟雾</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>烟雾传感器</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>红外传感器</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_inf_ray">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>无人员活动</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>振动传感器</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_shake">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>无振动</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_17">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_temp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>5</y>
       <width>40</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_18">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_humi">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>5</y>
       <width>40</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_19">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_lig">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>51</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>调光灯</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_pwm">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>客厅灯</string>
     </property>
    </widget>
    <widget class="QPushButton" name="light1_b">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="light2_b">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>卧室灯</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="light3_b">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风扇</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="text_log">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_16">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>煤气</string>
     </property>
    </widget>
    <widget class="QPushButton" name="light4_b">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>26</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>煤气已关</string>
     </property>
    </widget>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(224, 232, 250);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt;我们在您身边陪伴您&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt;陪您度过&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt;每一个&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt;美好的日子&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#c54848;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#c54848;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt;             &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#c54848;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#c54848;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c54848;&quot;&gt; by TeenDream&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
